README.md

```diff
@@ -593,6 +593,7 @@ Name | Website | Region
 [Skyrocket Ventures](/company-profiles/skyrocket-ventures.md) | https://www.skyrocketventures.com/ | Worldwide
 [SmartCash](/company-profiles/smartcash.md) | https://www.smartcash.cc/ | Worldwide
 [Smile.io](/company-profiles/smile.md) | https://smile.io | Worldwide
+[Smmile Digital](/company-profiles/smmile-digital.md) | https://smmile.com | Worldwide
```

Comment:
Alot of the roles are Singaporean only and as the URL for applying is Singaporean, please change `Worldwide` to `Singapore`